Dockerfile

```diff
@@ -15,4 +15,11 @@ ENTRYPOINT poetry run flask run --host=0.0.0.0
 FROM base as development
 WORKDIR /opt/todo_app
 ENV FLASK_DEBUG="true"
-ENTRYPOINT poetry run flask run --host=0.0.0.0
\ No newline at end of file
+ENTRYPOINT poetry run flask run --host=0.0.0.0
+
+FROM base as test
+WORKDIR /opt/todo_app

```

Comment:
(Is this being repeated a sign we'd be better putting it into the base stage?)
Edit: wait, it already is! No need to repeat in this stage in that case